x-pack/platform/packages/private/upgrade-assistant/server/src/es_version_precheck.ts

```diff
@@ -63,7 +62,8 @@ export const verifyAllMatchKibanaVersion = (allNodeVersions: SemVer[], majorVers
  */
 export const esVersionCheck = async (
   ctx: RequestHandlerContext,
-  response: KibanaResponseFactory
+  response: KibanaResponseFactory,
+  kibanaMajorVersion: number
```

Comment:
Its easier to pass in the major version number than to have it access a stateful Version class instance.